mvpa2/generators/resampling.py

```diff
@@ -203,3 +205,92 @@ def generate(self, ds):
     def __str__(self):
         return _str(self, str(self._amount), n=self._attr, count=self.count,
                     apply_selection=self._apply_selection)
+
+
+class LogExclusions(Node):
+    """Log excluded entries
+
+    Given a dataset with a boolean sample or feature attribute, return a
+    dataset with only those samples/features marked `True`.
```

Comment:
sounds very similar to what Splitter does (usually used after applying a Partitioner which would create such as sample attribute), but that one indeed wouldn't work across features